File: static_cdn/media_root/submissions/adiitrack/WINP.py
j=int(input())
str1="dheeraj"
str2="ranjeet"
sum1=0
sum2=0
for i in range(len(str1)):
	sum1=sum1+ord(str1[i]); 
for i in range(len(str2)):
	sum2=sum2+ord(str2[i]); 
temp1=0
temp2=0
for a in range(j):
	sum=0
	str3=input()
	
	for i in range(len(str3)):
		sum=sum+ord(str1[i]);    
		if sum>sum1 :
			break
		elif sum>sum2 :
			break
	if sum==sum1 :
		temp1+=1
	elif sum==sum2 :
		temp2+=1
if temp1>temp2 :
	print("dheeraj")
elif temp1<temp2 :
	print("ranjeet")
else: print("draw")

File: static_cdn/media_root/submissions/logicMachine/WINP.cpp
#include<iostream>
using namespace std;
int main()
{
	long int n,d=0,r=0;
	cin>>n;
	for(int i=0;i<n;i++)
	{
		string s;
		cin>>s;
		long int sum=0;
		for(int j=0;j<s.length();j++)
		{
			s[j]=s[j]-'a'+1;
			sum=sum+s[j];
		} 
		if(sum==51)
		d++;
		else if(sum==73)
		r++;
	}
	if(d>r)
	cout<<"dheeraj";
	else if(r>d)
	cout<<"ranjeet";
	else if(d==r)
	cout<<"draw";
}
